lib/l10n/intl_en_US.xml

```diff
@@ -3098,27 +3098,55 @@
     description="Text label for Drafts destination."
     >Drafts</string>
   <string
-    name="demoTwoPaneDualScreenLabel"
-    description="Option title for TwoPane demo in dual screen mode."
-    >Dual Screen</string>
+    name="demoTwoPaneFoldableLabel"
+    description="Option title for TwoPane demo in foldable mode."
```

Comment:
ensure descriptions make sense to a translator without any context, I would mention foldable devices in the description